oops/oops.go

```diff
@@ -6,10 +6,26 @@ import (
 	"bytes"
 	"fmt"
 	"io"
+	"os"
 	"runtime"
 	"strings"
 )
 
+var (
+	// filePrefixesToSkip is a set of prefixes of files which, when encountered when serializing
+	// an oops error, will skip appending the current frame and all subsequent frames in that stack.
+	// It is useful for truncating long call stacks to make logged oops errors more concise.
+	filePrefixesToSkip = make(map[string]struct{})
+)
+
+func init() {
+	envPrefixesToSkip := os.Getenv("OOPS_FILE_PREFIXES_TO_SKIP")
```

Comment:
This feels super heavy weight. Is there some other configuration pattern we could use?

Reply:
The one option I could think of is having a way of configuration oops via some externally exposed function (E.g., `Add/RemovePrefix` and doing an atomic load/store of values. Then we could get the set via loading the atomic. However this then requires configuration elsewhere in our services (which maybe is OK, we can do it in our grpcmiddleware instantiation, for instance). What are your thoughts?

Reply:
Changed to an atomic.Value with a function which allows overwriting the atomic value. I think this is sufficient for use cases, since configuration of stack traces can be done in a single caller per-program.